.evg.yml

```diff
@@ -438,16 +453,6 @@ functions:
         remote_file: mongosqltranslate/${lib_prefix}mongosqltranslate-v${LIBMONGOSQLTRANSLATE_VER}-${platform}-${arch}.${ext}
         bucket: translators-connectors-releases
 
-  "fetch mongosqltranslate windows":
-    - command: s3.get
-      params:
-        working_dir: mongo-jdbc-driver
-        aws_key: ${release_aws_key}
-        aws_secret: ${release_aws_secret}
-        local_file: mongo-jdbc-driver/src/main/resources/x86_64/windows/mongosqltranslate.dll
-        remote_file: mongosqltranslate/mongosqltranslate-v${LIBMONGOSQLTRANSLATE_VER}-win.dll
-        bucket: translators-connectors-releases
-
```

Comment:
Since `fetch libmongosqltranslate` uses `platform` for the local file path and in the name of the remote file, the library will now be put in `mongo-jdbc-driver/src/main/resources/x86_64/win/` instead of `mongo-jdbc-driver/src/main/resources/x86_64/windows/`. Will this be a problem?